lib/packer.js

```diff
@@ -15,3 +27,28 @@ function encodeDate(obj) {
 function decodeDate(buf) {
   return new Date(buf.toString('utf8'));
 }
+
+function jsonFormat() {
+  var packer = {
+    encode: function(value) {
+      return JSON.stringify (value);
```

Comment:
The default JSON representation of Buffers is rather inefficient:

```
> JSON.stringify(new Buffer([1,2,3]))
'{"type":"Buffer","data":[1,2,3]}'
```

It makes me wonder wether we should use a different representation and base64-encode the data? For example:

```
{ "type": "Buffer", "data": "AQID" }
```